tests/Aspire.EndToEnd.Tests/Aspire.EndToEnd.Tests.csproj

```diff
@@ -2,6 +2,7 @@
 
   <PropertyGroup>
     <TargetFramework>$(NetCurrent)</TargetFramework>
+    <ProductVersion>8.2.0</ProductVersion>
```

Comment:
Why do we have to hard-code this here and below? Why doesn't `PackageVersion` just work?

Reply:
This one can theoretically be $major.$minor, but below (meaning in the helix packaging props) we can't use PackageVersion because PackageVersion is evaluated for the project being built, which for our test projects won't be a stable prerelease version (meaning it would look like `8.2.0-ci` as opposed to `8.2.0`) The real fix here, is that to get these versions they should be built on-the-fly by making a call onto each one of these packages projects, and asking which version are those producing in order to get a full accurate set. The file below is instead doing the "lazy" approach of "guessing" what the right versions are, which is not accurate when producing stable versions as we have some packages we are stabilizing and some we are keeping in preview.

Reply:
Oh, it is assuming that the current project's (i.e. the test project's) Version is the same as the built package's versions. Yeah, this seems like something to fix.